[PPJA] Even More Recipes/[JA] Even More Recipes - Meat/Objects/Bangers Beans and Mash/object.json

```diff
@@ -47,7 +47,7 @@
     {
         "es": "Bangers Beans and Mash",
         "ko": "소시지 콩과 으깬 감자",
-        "de": "Bohnenpüree",
+        // "de": "",
```

Comment:
Using this as an example, the current version has `"de": "Bohnenpüree",` while your commit removes the German translation.

Reply:
I see...

Reply:
Is there a precedent for older files being sent here instead of the newer ones, or a merge with previous pull requests etc? I followed the guide and checked many times that the local version I was working on was the newest version

Reply:
For example, I deleted my local files a few times because it kept sending the old files I did a week ago instead of the ones I did yesterday

Reply:
If you're working off of the same branch as the one you did a week ago it'll keep sending the old version.

I'd try making a new branch with the changes and opening a new PR from that branch. I'm not super familiar with github myself so if you need more help I can try to bring someone in who understands github a bit more to help out.